src/content/effective-dart/documentation.md

```diff
@@ -231,19 +232,17 @@ It's better to say nothing
 than waste a reader's time telling them something they already know.
 
 
-### PREFER starting function or method comments with third-person verbs
+### PREFER starting comments of a function or method with third-person verbs if its main purpose is a side effect
 
 The doc comment should focus on what the code *does*.
 
 <?code-excerpt "docs_good.dart (third-person)"?>
 ```dart tag=good
-/// Returns `true` if every element satisfies the [predicate].
-bool all(bool predicate(T element)) => ...
```

Comment:
This is precisely the kind of function this change is about, so it's not a "good" example of using a verb-documentation.
(It's also the example people keep pointing to as why they use "Returns".)

Reply:
I totally agree with you. I have a slightly different approach to dealing with these. There is an elegant way of dealing with function summaries in the GoogleSQL reference docs that I feel we should pull over here. Example:

https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions

Reply:
Resolving since I have a better idea of how things map with each other now.